examples/obsidian-dataview/README.md

```diff
@@ -0,0 +1,192 @@
+- `from "characters"`: This tells Dataview to pull the data from all markdown files located in the `/characters` folder in your Obsidian vault.
+- `sort name asc`: This command sorts the data based on the "name" field, in ascending order.
+
+After you click somewhere outside of this code block, you should see the following table:
+
+![Table](table1.png)
+
+## Step 4: Add more metadata about characters
+
+We can continue to add more information about our characters. For example, let's add creatures each character owns:
+
+```md=
+---
+name: Harry Potter
+...
+creature: Hedwig (Owl)
+---
+
+...
+
+---
+
+name: Hermione Granger
+...
+creature: Crookshanks (Cat)
+---
+
+...
+
+```
+
+Now, if we want to show this field in our Dataview table, we need to add it to the first line of our dataview code block:
+
+````md
+```dataview
+table name, house, status, creature
+from "characters"
+sort name asc
+```
+````
+
+After clicking somewhere outside the code block, you should see an updated table that includes the column `creature`.
+
+![Table](table2.png)
+
+## Step 5: Enrich content with images and links
+
+To enhance your catalog, you can add images of characters, and links to other characters in your markdown content.
+
+#### Add images
+
+To add an image of a character, you can just copy it to clipboard and paste it directly in your markdown file. Obsidian will automatically save the file to the root of your vault and create a link to it in your content, so you'll end up with a link like this one:
+
+```md
+![[Pasted image 20230525212302.png]]
+```
+
+Which will render as:
+
+![](harry.png)
+
+#### Add links to other pages
+
+Obsidian allows you to create links between different notes (or in this case, characters). To add a link to another character, use the double bracket `[[]]` syntax.
+
+For example, if you're writing about Harry Potter and want to mention that he is friends with Hermione, you can link to Hermione's markdown file like this:
+
+```md
+He is a friend of [[Hermione]].
+```
+
+When you click on Hermione's name in the rendered link, Obsidian will take you to Hermione's file in your vault. If you hover over it, you'll see a preview of Hermione's file:
+
+![Link preview](link-preview.png)
+
+## Summary
+
+And there you have it! We've walked through the process of creating a personal data catalog using markdown files, the Obsidian application, and its Dataview plugin. We've transformed scattered data into a well-organized, easy-to-navigate, and visually pleasing catalog. Whether you've used our example of a Harry Potter character catalog or applied these steps to a different topic of your choosing, we hope you've found this tutorial helpful and empowering.
+
+Happy data cataloging!
+
+---
+
+You can find the vault created in this tutorial [here](https://github.com/datopian/markdowndb/tree/main/examples/dataview).
```

Comment:
I think this link is going to be broken even after we merge

Reply:
I'm going to fix this now.